daemons/controld/controld_execd.c

```diff
@@ -2697,8 +2697,7 @@ process_lrm_event(lrm_state_t *lrm_state, lrmd_event_data_t *op,
             break;
 
         case PCMK_LRM_OP_DONE:
-            do_crm_log((op->interval_ms? LOG_INFO : LOG_NOTICE),
-                       "Result of %s operation for %s on %s: %d (%s) "
+            crm_notice("Result of %s operation for %s on %s: %d (%s) "
```

Comment:
This has the advantage for when libqb with the linker section enabled
is used, since deferring to `qb_log_from_external_source` was imposing
inefficiencies (the kind that would be easily resolved with a macro
I proposed in the iterations over the recent security fixes but which
Ken refused to accept; what I want to say, perhaps there will be enough
of remaining use cases -- minus this one -- with conditional priority
that such macro would still be justified).